validate.go

```diff
@@ -106,6 +107,29 @@ func checkMounts(mounts []rspec.Mount, rootfs string) {
 	}
 }
 
+func checkHooks(process rspec.Hooks, rootfs string) {
```

Comment:
@wking  change `process` to `hooks` is fine.
`hooks are in the host mount namespace` I'm not sure what it means.
Do you mean hooks is not in OCI bundle?